io-ts/package.json

```
@@ -0,0 +1,19 @@
+{
+  "name": "io-ts",
+  "amdName": "hookformResolversIoTs",
+  "version": "1.0.0",
+  "private": true,
+  "description": "React Hook Form validation resolver: io-ts",
+  "main": "dist/io-ts.js",
+  "module": "dist/io-ts.module.js",
+  "umd:main": "dist/io-ts.umd.js",
+  "source": "src/index.ts",
+  "types": "dist/index.d.ts",
+  "license": "MIT",
+  "peerDependencies": {
+    "react-hook-form": "^7.0.0",
+    "@hookform/resolvers": "^2.0.0",
+    "io-ts": "^2.2.13",
```

```suggestion
    "io-ts": "^2.0.0",
```

I'm new to `io-ts`, I read the changelog and I think it safes to use `^2.0.0` can you confirm? Same question for `fp-ts`

---

About `io-ts`, I don't use a lot of stuff from it, so yes, 2.0.0 should be way enough. Concerning `fp-ts`, I use `ReadonlyArray.last` for instance that was introduced in 2.5.0. I'll check it more carefully. Any thanks a lot for your feedbacks @jorisre !

---

So it's ok for `io-ts` @ `^2.0.0` but for `fp-ts` I can only go to [^2.7.0](https://github.com/gcanti/fp-ts/releases/tag/2.7.0) as I use `ReadonlyArray.Foldable`